src/test/java/org/imsglobal/caliper/ri/events/AssessmentEventSubmittedTest.java

```diff
@@ -0,0 +1,153 @@
+import org.imsglobal.caliper.entities.agent.Status;
+import org.imsglobal.caliper.entities.resource.Assessment;
+import org.imsglobal.caliper.entities.resource.Attempt;
+import org.imsglobal.caliper.entities.session.Session;
+import org.imsglobal.caliper.events.AssessmentEvent;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+
+import static com.yammer.dropwizard.testing.JsonHelpers.jsonFixture;
+import static org.imsglobal.caliper.ri.events.HMHConstants.*;
+import static org.imsglobal.caliper.ri.events.HMHConstants.ACTIVITY_REF_ID;
+import static org.imsglobal.caliper.ri.events.HMHConstants.APP_NAME;
+import static org.imsglobal.caliper.ri.events.HMHConstants.BASE_IRI;
+import static org.imsglobal.caliper.ri.events.HMHConstants.BASE_URN;
+import static org.imsglobal.caliper.ri.events.HMHConstants.STUDENT_NAME;
+import static org.imsglobal.caliper.ri.events.HMHConstants.STUDENT_USER_REF_ID;
+
+@Category(org.imsglobal.caliper.UnitTest.class)
+public class AssessmentEventSubmittedTest {
+    private JsonldContext context;
+    private String id;
+    private Person actor;
+    private Assessment object;
+    private Attempt generated;
+    private SoftwareApplication edApp;
+    private CourseSection group;
+    private Membership membership;
+    private Session session;
+    private AssessmentEvent event;
+
+    @Before
+    public void setUp() throws Exception {
+        context = JsonldStringContext.getDefault();
+
+        String UNIQUE_EVENT_ID = "bd232c3c-d86c-45b6-b2eb-3d7688ceeee5";
+        id = BASE_URN + UNIQUE_EVENT_ID;
+
+        actor = Person.builder().id(BASE_URN.concat(STUDENT_USER_REF_ID)).name(STUDENT_NAME).build();
+        Person assignee = Person.builder().id(actor.getId()).coercedToId(true).build();
+
+        object = Assessment.builder()
+            .id(BASE_URN.concat(ACTIVITY_REF_ID))
+            .name(APP_NAME)
+            .maxAttempts(1)
+            .maxSubmits(1)
+            .maxScore(25.0)
+            .version("1.0")
+            .build();
+
+        String UNIQUE_ATTEMPT_ID = "f22de1be-8c76-4e65-9af7-50aafcb1c470";
+        generated = Attempt.builder()
+            .id(BASE_URN.concat(UNIQUE_ATTEMPT_ID))
+            .assignable(Assessment.builder().id(object.getId()).coercedToId(true).build())
+            .assignee(assignee)
+            .count(1)
+            .dateCreated(new DateTime(2016, 11, 15, 10, 15, 0, 0, DateTimeZone.UTC))
+            .startedAtTime(new DateTime(2016, 11, 15, 10, 15, 0, 0, DateTimeZone.UTC))
+            .build();
+
+        edApp = SoftwareApplication.builder().id(BASE_IRI.concat(APP_NAME)).coercedToId(true).build();
+
+        membership = Membership.builder()
+            .id(actor.getId())
```

Comment:
I will fix this once we figure out what it should be https://github.com/hmhco/caliper-java/pull/2/files#diff-a9dd32089be2e8e5fc7f77a392a5c022R104